src/consumption.cpp

```diff
@@ -185,7 +185,7 @@ std::pair<int, int> player::fun_for( const item &comest ) const
         }
     }
 
-    return std::pair<int, int>( fun, fun_max );
+    return { ( int )fun, ( int )fun_max };
```

Comment:
Don't use C-style cast - use `static_cast< int >` instead.

Reply:
I generally prefer that, though I didn't see any in the existing code, not that I checked very hard.